core/storage/ipfs/merkledag/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+find_package(Protobuf REQUIRED)
+set(PB_SCHEME "merkledag.proto")
```

Comment:
Maybe it's ok for now, but pls consider moving to the way protobuf used [here](https://github.com/soramitsu/libp2p/blob/942d3cbe4387af2020ae11f8abbdae8e6a2b0448/cmake/functions.cmake#L93) 
and [here](https://github.com/soramitsu/libp2p/blob/38cf8c4f99617b6076f8ab978bd8acce7af61329/src/crypto/protobuf/CMakeLists.txt#L6)